src/components/data/data-utils.js

```diff
@@ -273,6 +274,7 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
   const isInvalid = isFileImportCurrMode === isFileImportLastUsedMode && file && !match
   const newEntityType = match?.[1]
   const currentFile = isFileImportCurrMode === isFileImportLastUsedMode ? file : undefined
+  const containsNullValues = fileContents.match('\t\t')
```

Comment:
This doesn't identify null values in the last column of a TSV.

Reply:
😭 nice catch

Reply:
Perhaps:
```javascript
fileContents.match(/^\t|\t\t+|\t$|\n\n+/gm)
```

Reply:
Thank you for saving me the regex-headache. That seemed to do the trick.

Resolved in e4484f2688c78cc7e1477f1717cc54f71027a9fe